features/base/participants/actions.js

```diff
@@ -250,3 +250,24 @@ export function remoteParticipantJoined(id, user = {}) {
         }
     };
 }
+
+/**
+ * Removes local participant.
+ *
+ * @returns {Function}
+ */
+export function removeLocalParticipant() {
+    return (dispatch, getState) => {
+        let localParticipant = getState()['features/base/participants']
+            .find(p => p.local);
+
+        if (localParticipant) {
+            dispatch({
+                type: PARTICIPANT_REMOVED,
```

Comment:
According to our coding style, we should use 1 name for 1 abstractions (even across multiple projects). At the moment we have, (1) participantLeft and PARTICIPANT_REMOVED and (2) participantJoined and PARTICIPANT_ADDED. Because lib-jitsi-meet's events say LEFT and JOINED, the action types should please be PARTICIPANT_LEFT and PARTICIPANT_JOINED.

Additionally, I noticed that participant and user are used interchangeably. I see that lib-jitsi-meet uses these 2 names for one and the same abstractions but we should abide by our coding style and use 1 name. Participant has been choses by the state features/base/participants so we shouldn't use user. I saw the word user in function arguments, jsdcos. Please make sure to use participant everywhere.